corehq/apps/style/templates/style/includes/analytics_kissmetrics.html

```diff
@@ -26,6 +26,7 @@
      *      Default is 2000 milliseconds.
      */
     function kmqPushSafe(args, timeout) {
+        console.log("called kissmetrics with", arguments);
```

Comment:
I think we just removed the `console.logs` for GA, since they were getting spammy. I would suggest we don't keep this here, since we should actually be able to see outgoing requests in the the network tab in dev tools.